core-feature-pack/src/main/resources/modules/system/layers/base/org/jboss/as/cli/main/module.xml

```diff
@@ -35,6 +35,25 @@
     </resources>
 
     <dependencies>
+        <!-- JAXP default dependencies. DO NOT REMOVE!!!
+                   These are loaded by the JAXP redirect facility in jboss-modules when this module is used
+                   as a boot module (this is the case for server processes in domain mode). The purpose is
```

Comment:
Can you provide more details about how this module is used as a boot module for server processes in domain mode? It sounds strange to me that this module is relevant to a server process, except perhaps if the server is embedded.